packages/components/src/internal/components/assay/utils.tsx

```diff
@@ -59,3 +67,56 @@ export function getServerFilePreview(file: string, numLinesToInclude: number): P
         });
     });
 }
+
+function getAssayImportSuccessMsg(
+    response: AssayUploadResultModel,
```

Comment:
If all we really need here is the `runId` then maybe consider handing in just that instead of the entire `AssayUploadResultsModel`.